auction-server/src/auction.rs

```diff
@@ -985,12 +1006,31 @@ pub async fn handle_bid_svm(
 
     let submit_bid_instruction =
         verify_submit_bid_instruction_svm(chain_store, bid.transaction.clone())?;
-    let (_bid_amount, _permission_key) = extract_bid_data_svm(
+    let (bid_amount, permission_key) = extract_bid_data_svm(
         store.express_relay_svm.clone(),
         bid.transaction.message.static_account_keys(),
         submit_bid_instruction,
     )?;
 
-    // TODO implement this
-    Err(RestError::NotImplemented)
+    // TODO simulate bid
+
+    let bid_id = Uuid::new_v4();
```

Comment:
there is some duplicate logic creating this core_fields in evm and svm which we can refactor